astropy/utils/data.py

```diff
@@ -36,7 +36,7 @@
     'get_readable_fileobj',
     'get_pkg_data_fileobj', 'get_pkg_data_filename',
     'get_pkg_data_contents', 'get_pkg_data_fileobjs',
-    'get_pkg_data_filenames',
+    'get_pkg_data_filenames', "get_pkg_data_path",
```

Comment:
Nitpick: Have the quotes be consistent.
```suggestion
    'get_pkg_data_filenames', 'get_pkg_data_path',
```